OpenRA.Mods.Common/Widgets/Logic/CreditsLogic.cs

```diff
@@ -39,7 +40,9 @@ public CreditsLogic(Widget widget, ModData modData, Action onExit)
 				onExit();
 			};
 
-			engineLines = ParseLines("AUTHORS");
+			engineLines = ParseLines(
```

Comment:
Again, please use write this, specifically, as `engineLines = ParseLines(File.OpenRead(Platform.ResolvePath("./AUTHORS")));`. This `./` prefix is important for resolving the absolute game path.

Reply:
Thought that was implicit. Done.

Reply:
This needs to be on a single line - splitting it over 3 breaks the coding conventions that are enforced by the continuous integration server, breaking the build.

Reply:
I see. Done.